spec/models/lesson_spec.rb

```diff
@@ -118,6 +118,29 @@
     end
   end
 
+  describe '#recently_added?' do
+    context 'when the lesson is added today' do
```

Comment:
Nit: _Extremely_ minor, but using a consistent tense, usually past tense, makes the test readability flow a tiny bit better.

```suggestion
    context 'when the lesson was added today' do
```